Unknown Network Should Yield 404

Right now I get a 500:
```
AssertionError: Network my_id not registered
    at getNetwork (/usr/src/app/server/lib/network-registry.js:15:3)
    at /usr/src/app/server/api/tunnel.js:14:21
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/usr/src/app/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at /usr/src/app/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/usr/src/app/node_modules/express/lib/router/index.js:330:12)
    at next (/usr/src/app/node_modules/express/lib/router/index.js:271:10)
    at Function.handle (/usr/src/app/node_modules/express/lib/router/index.js:176:3)
```

--- a/server/api/tunnel.js
+++ b/server/api/tunnel.js
@@ -12,6 +12,9 @@ export default function () {
     assert(netloc);
 
     const network = getNetwork(req.networkId);
+    if (!network) {
+      return res.status(404).send('network not found');
+    }
 
 
     const tunnelServerHost = req.header('host').replace(/:.*$/, '');

--- a/server/lib/network-registry.js
+++ b/server/lib/network-registry.js
@@ -1,5 +1,3 @@
-import assert from 'assert';
-
 import Network from './network';
 import { log, warn } from './log';
 
@@ -11,9 +9,7 @@ const networkRegistry = {};
  * @return {[type]} [description]
  */
 export function getNetwork(networkId) {
-  const network = networkRegistry[networkId];
-  assert(network, `Network ${networkId} not registered`);
-  return network;
+  return networkRegistry[networkId];
 }
 
 export function unregisterNetwork(networkId) {

